3_Parameter_with_callback/README.md

```diff
@@ -26,7 +26,7 @@ If all went fine under **devices** tab on Live LiveObjects portal you should see
 **Is device sending data:**<br>
 Under data tab on LiveObjects portal you should see messages from stream identified by modem IMEI, along with values *{ "uptime": xxxxx }*
 
-## Send command
+## Send parameter

```

Comment:
better 'Update parameter'

Reply:
Agree! Will fix in some of next pull requests as I see you already merged it.